agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -17,7 +17,7 @@ pbench_bin="`cd ${script_path}/..; /bin/pwd`"
 group=default
 dir="/tmp"
 sysinfo_opts_default=( block libvirt kernel_config sos topology )
-sysinfo_opts=${sysinfo_opts_default[*]}
+sysinfo_opts="${sysinfo_opts_default[*]} ara"
```

Comment:
@portante if we add ara to the default options, its safe but there won't be any difference between all and default options. If I am not wrong the idea is to allow users to choose whether to collect the default set or collect all right?